### ISSUE TITLE
Out of date - what do we do with this?

### ISSUE THREAD

**Issue by iteles** [dwyl/linky#12]
This repo doesn't reflect our [current tech stack](http://www.github.com/dwyl/technology-stack), includes a fair few things we don't advocate (babel and webpack for example, despite this being a react setup, is still misleading) and no one has picked up the baton in the last year as well as being superseded by upcoming work.

What should we do with this repo? Add a big 'DEPRECATED' banner across the top? Add deprecated to the title of the repo?

**Comment by nelsonic** [comment]
@iteles I'm updating https://github.com/dwyl/technology-stack but only after at least a handful of our people have some Elixir Experience ... `setup` will be updated accordingly. 👍

**Comment by nelsonic** [comment]
Obsolete. ✅ 
Repurpose. ♻️
